### ISSUE TITLE
Annual Status Not Updating as "Downloaded"

### ISSUE THREAD

**Issue by bguidotti** [evilhero/mylar#1579]
I have an issue where Annuals will not show as Downloaded status when present in the comic folder.
They initially show as "Skipped" status.
When I click the checkbox to select the issue(s), and then click the dropdown "Downloaded" in the 'Mark selected annuals as xxx selected annuals' box, nothing happens.
Watching the status in real-time in Python, nothing seems to kick-off.
However, if I click the "Archived" status, Mylar checks the folder, and then marks them as "Downloaded."
If I don't click "Downloaded" the first time, it will mark them as "Archived."  If I then click "Downloaded," it will change them to "Downloaded" status.

Hopefully this is clear; I know it sounds confusing.
Basically, I just want Mylar to check the Annuals and mark the status correctly when in the same folder as the other comics on first scan and/or when I do a re-check files or refresh comic.

Thanks!
-Bruce

**Comment by evilhero** [comment]
Because you can't mark an issue as downloaded as an initial state, it can only be put in that state by mylar.

It will also not mark an issue as archived if the file is present within the directory (the purpose of the archive status is to indicate the comic is present, just not in the given directory as it was moved /deleted by the user on purpose). 

Put mylar into verbose mode and do a recheck files on a problem series. Then paste the debug log here - as long as you have annual integration enabled and the series has already been refreshed to show the annuals it should have no problems marking them in the correct state (it works for me currently)
